NAMESPACE

```diff
@@ -7,13 +7,9 @@ S3method(format,travis_user)
 S3method(print,travis)
 export(auth_github)
 export(auth_travis)
-export(github_add_key)
-export(github_create_pat)
-export(github_create_repo)
-export(github_info)
+export(browse_travis)
 export(github_repo)
-export(new_github)
-export(travis_browse)
+export(travisHTTP)
```

Comment:
If we export this I believe we need a nicer name than that. Can we move current `?travis` to `?"travis-pkg"` and rename this function to `travis()` ?

Reply:
rename done, added `?travis-package`